hooks/pre-push

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+# Use this file to prevent pushes to origin
+# Based on https://gist.githubusercontent.com/stefansundin/d465f1e331fc5c632088/raw/pre-push-2
+#
+# This is not installed by install_git_settings.sh, to turn this on, use a similar command
+# as the one there that creates a symlink in .git/hooks
+
+ORIGIN="$1"
+REMOTE_URL="$2"
+
+if [[ "$REMOTE_URL" == *"origin/"* ]]; then
```

Comment:
I think this needs to specifically mention mobilecoinofficial, my local checkout names `jcape/mobilecoin.git` asd `origin`, and `mobilecoinofficial/mobilecoin.git` as `upstream`

Reply:
okay I'll play with it

i'm not sure but I think there might be no way to make it work for every person, it may depend on how you created your local checkout / how you configured remotes

https://git-scm.com/docs/githooks#_pre_push